email/testing.rst

```diff
@@ -22,7 +22,7 @@ Start with a simple controller action that sends an email::
 
         $mailer->send($message);
 
-        return $this->render(...);
+        return $this->render();
```

Comment:
This could probably be replaced with `return new Response();` instead? I guess the point here was "return some kind of response".

Reply:
I am :+1: for return new Response here instead of „...“

Reply:
At the end I replaced this line by `// ...` because it's not related or needed for the rest of the article.